packages/dependency-extraction-webpack-plugin/lib/util.js

```diff
@@ -71,6 +69,8 @@ function defaultRequestToHandle( request ) {
 	if ( request.startsWith( WORDPRESS_NAMESPACE ) ) {
 		return 'wp-' + request.substring( WORDPRESS_NAMESPACE.length );
 	}
+
+	return undefined;
```

Comment:
Fun 😄 

I can't say I really understand why the implicit return isn't enough to satisfy an `undefined` return type.

Reply:
Thanks for calling that out. A return type of `undefined` requires a return (depending on your setup). `void` will be better, it allows both explicit `return undefined` and omission.

I've changed this to use `void`.